pytorch_lightning/loops/dataloader/prediction_dataloader_loop.py

```diff
@@ -0,0 +1,149 @@
+from typing import Any, List, Optional, Sequence, Union
+
+from deprecate.utils import void
+from torch.utils.data import DataLoader
+
+import pytorch_lightning as pl
+from pytorch_lightning.loops.dataloader.dataloader_loop import DataLoaderLoop
+from pytorch_lightning.loops.prediction_loop import PredictionLoop
+from pytorch_lightning.plugins import DDPSpawnPlugin
+from pytorch_lightning.utilities.exceptions import MisconfigurationException
+from pytorch_lightning.utilities.types import _PREDICT_OUTPUT
+
+
+class PredictionDataLoaderLoop(DataLoaderLoop):
```

Comment:
I think it'd look nicer if this was just called `PredictLoop`.

IMO the fact that it subclasses `DatLoader` loop is a detail.

Just as your `class Square(Rectangle)` wouldn't be named `SquareRectangle`

This also applies to the existing loops

Reply:
sounds good, let me rename all loops together  in a new PR where we can also decide on the package structure how we want to organize files?

Reply:
Sure

Reply:
#8077